_overviews/scala3-book/methods-most.md

```diff
@@ -383,23 +391,6 @@ See the [Extension methods section][reference_extension_methods] of this book, a
 
 
 
-## Even more
-
-There’s even more to know about methods, including how to:
-
-- Call methods on superclasses
-- Define and use by-name parameters
-- Write a method that takes a function parameter
-- Create inline methods
-- Handle exceptions
-- Use vararg input parameters
-- Write methods that have multiple parameter groups (partially-applied functions)
-- Create methods that have generic type parameters
-
-See the [Reference documentation][reference] for more details on these features.
-
-
-
```

Comment:
I agree placeholder text isn't helpful unless it is meant to be expanded in place. I'm not actually sure if this is meant to be closer to the Scala tour than an overview.

Reply:
and?